pinot-core/src/main/java/com/linkedin/pinot/core/plan/maker/InstancePlanMakerImplV2.java

```diff
@@ -27,6 +27,7 @@
 import com.linkedin.pinot.core.plan.PlanNode;
 import com.linkedin.pinot.core.plan.SelectionPlanNode;
 import com.linkedin.pinot.core.query.config.QueryExecutorConfig;
+import com.linkedin.pinot.pql.parsers.pql2.ast.IntegerLiteralAstNode;
```

Comment:
Good catch. I think the reason might be when I type the Integer.MAX_VALUE, I mis-auto-import the wrong class.